scripts/start-local-development.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```

Comment:
What is the use of this file?

Reply:
It polls the code base for changes, re-builds, and deploys to local maven continuously using Gradle. You can hop back and forth to the core Android project more easily and make changes (still have to press "Gradle Sync" in the core project though).